### PR TITLE
Update channel map to incorporate default track + safest risk level + version

### PR DESCRIPTION
**Pull request by therealjuan**
1. Pull the branch or run the demo server
2. Open any snap detail page
3. Channel map actions should be `[ default_track/safest_risk_level  version_number ] [ Install ]` on every snap page

**NOTE:** Some snaps to test with `[node](https://snapcraft.io/node)` , `[skype](https://snapcraft.io/skype)`, `[blender](https://snapcraft.io/blender)`

🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉

### CODE REVIEW

File: webapp/store/logic.py [hunk 1]
```diff
@@ -285,3 +285,22 @@ def get_confinement(channel_map, track, risk):
...
+
+
+def get_version(channel_map, track, risk):
+    """Get the version for a channel
+
+    :param channel_map: Channel map list
+    :param track: The track of the channel
+    :param risk: The risk of the channel
+
+    :returns: The confinement
```
Comment (tbille) [inline webapp/store/logic.py]: bad copy paste :P

File: webapp/store/logic.py [hunk 2]
```diff
@@ -285,3 +285,22 @@ def get_confinement(channel_map, track, risk):
...
+    """Get the version for a channel
+
+    :param channel_map: Channel map list
+    :param track: The track of the channel
+    :param risk: The risk of the channel
+
+    :returns: The version
+    """
+    for arch in channel_map:
+        if arch in channel_map and track in channel_map[arch]:
```
Comment (bartaz) [inline webapp/store/logic.py]: Isn't `if arch in channel_map` always true because you are iterating over `for arch in channel_map`?

**Comment by therealjuan** [comment]
The underlying problem of what you describe is a lack of education towards basic concepts related to snaps somewhere in snapcraft.io: risk levels, channels... specially in cases where snaps have no stable releases. 

Let's open another issue to work on this as a `How it works` could be interested to include similar to what maas.io has to explain some concepts

File: webapp/store/logic.py (hunk 2 above)
Comment (therealjuan) [reply webapp/store/logic.py]: 👍 just updated

Comment (therealjuan) [reply webapp/store/logic.py]: @tbille mentioned that these two functions to get confinement and get version can do with some refactoring

**Pull request merged**